corehq/apps/domain/templates/domain/admin/info_basic.html

```diff
@@ -6,6 +6,17 @@
 {% block js-inline %} {{ block.super }}
 <script type="text/javascript">
     $(function() {
+        // Update project dropdown in page header with potentially new project name
+        var hr_name = $('#id_hr_name').val(),
+            $projectsMenu = $('#nav-projects-menu'),
+            currentHref = $projectsMenu.find(".btn-project-active").attr("href");
+        _.each($projectsMenu.find("a"), function(link) {
+            var $link = $(link);
+            if (currentHref.startsWith($link.attr("href"))) {
```

Comment:
what's with the `startsWith` here?  I assume you're looking for the link corresponding to the domain that was just changed, shouldn't the two urls be equal?  If I update the display name for my domain "ethan", will this cause "ethan-test" to show up wrong?